src/server.rs

```diff
@@ -427,6 +431,10 @@ fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
 		credential_gateway.run().await;
 	});
 
+	tokio::spawn(async move {
+		inclusion_proof.run().await;
```

Comment:
i see in the future, we might move inclusion proof in its own thread / daemon (e.g in `notaryd`) though for now good if we keep it like this.